modules/affile/file-signals.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Shikhar Vashistha
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
```

Comment:
We should change the license of this file to LGPL in my opinion.

For example:
https://github.com/syslog-ng/syslog-ng/blob/master/lib/cache.h#L4-L20

Please add this file to the copyright checker script too:
https://github.com/syslog-ng/syslog-ng/blob/master/tests/copyright/policy#L119

You can test it by running `make check-copyright`.

Reply:
Updated :+1: